dashboard-api/Makefile

```diff
@@ -0,0 +1,60 @@
+default: help
```

Comment:
Love having a Makefile. 

Seems like there are two versions, one where we use docker compose and one where we just run `go test`. 

The docker compose, I assume, is for running the DB? But `go test` doesn't need the DB, or...?

Reply:
Yes this was some confusion for me as well as you have a `docker-compose.test.yaml`, but you don't actually need a test. I added the target as it was in the test.sh file

Reply:
Let me know if you want me to delete the docker-compose stuff

Reply:
Yeah, if it's not being used, clean it out!

Reply:
Did a clean up of unnecessary files as well.